modules/@apostrophecms/template/lib/bundlesLoader.js

```diff
@@ -1,4 +1,5 @@
 const { stripIndent } = require('common-tags');
+const { reBundle } = require('../../../../lib/bundle-helpers');
```

Comment:
Please follow the Apostrophe pattern of methods so that extension/override at project level or in an `improve` module is possible. If you want to break them out to separate files you can do that with `...` and `require` inside `methods`.